unison-core/src/Unison/Names.hs

```diff
@@ -145,10 +148,14 @@ fuzzyFind query names =
                          query
           )
 
-termReferences, typeReferences, allReferences :: Names -> Set Reference
+termReferences, typeReferences :: Names -> Set Reference
 termReferences Names{..} = Set.map Referent.toReference $ R.ran terms
 typeReferences Names{..} = R.ran types
-allReferences n = termReferences n <> typeReferences n
```

Comment:
Trying to clean up some of these areas where we end up with references of ambiguous origin.